invenio_oaiserver/ext.py

```diff
@@ -68,8 +68,8 @@ def register_signals(self):
                                                      weak=False)
         records_signals.before_record_update.connect(self.update_function,
                                                      weak=False)
-        if self.app.config['OAISERVER_REGISTER_SET_SIGNALS']:
-            self.register_signals_oaiset()
+        #  if self.app.config['OAISERVER_REGISTER_SET_SIGNALS']:
```

Comment:
The hook functions are also creating/destroying the percolators.
It means that at some point you can potentially have more sets than percolators.
I think make sense to implement something similar to `reindex` for indices to be able to sync the oaisets with the percolator in ES. WDYT?